marshaler.go

```diff
@@ -89,8 +89,12 @@ func (enc *Encoder) SetIndentTables(indent bool) *Encoder {
 	return enc
 }
 
-// SetMarshalJsonNumbers forces the encoder to serialize `json.Number` as a float or integer
-// instead of relying on TextMarshaler to emit a string.
+// SetMarshalJsonNumbers forces the encoder to serialize `json.Number` as a
+// float or integer instead of relying on TextMarshaler to emit a string.
+//
+// *Unstable:* This method does not follow the compatiblity guarnatees of
```

Comment:
small typo: s/guarnatees/guarantees/

Reply:
🤦🏻‍♂️ good catch, thank you https://github.com/pelletier/go-toml/pull/927